### DISCUSSION TITLE
[Suggestion] Toning down the parallax on the islands

### DISCUSSION THREAD

**Discussion by fromulus** [Monika-After-Story/MonikaModDev#8246] [issue discussions]
So, don't get me wrong, the effect's really nice, but I would argue it needs some more tweaking, as it currently looks a little _too_ unrealistic, or at least enough to bother me.

For example, the island in the back feels like it's following the mouse way too much.
I get that impression with the island on the front too (though to a lesser extent), and it feels all the more unimmersive considering those are meant to be proper landmasses, even if flying ones.

Not sure if I'm the only one feeling that way, but I'll just leave the issue here for discussion, at least.

**Comment by Booplicate** [comment]
I'm not entirely sure I understand you. The idea behind the changes wasn't to create something ultra realistic, just add some life to the islands, make them more interesting. But even then, why do you think it's unrealistic? The further the islands are, the less they move, so technically the parallax effect is correct.

>  it feels all the more unimmersive considering those are meant to be proper landmasses, even if flying ones.

The effect isn't being affected by the object mass or size, only the distance to it.

**Comment by ThePotatoGuy** [comment]
moving to issue discussion - we are not likely to change this at this time.
